rai/node/wallet.cpp

```diff
@@ -386,7 +386,11 @@ std::vector<rai::account> rai::wallet_store::accounts (MDB_txn * transaction_a)
 void rai::wallet_store::initialize (MDB_txn * transaction_a, bool & init_a, std::string const & path_a)
 {
 	assert (strlen (path_a.c_str ()) == path_a.size ());
-	auto error (mdb_dbi_open (transaction_a, path_a.c_str (), MDB_CREATE, &handle));
+	std::stringstream ids_path;
+	ids_path << path_a << "_send_action_ids";
+	auto error (0);
+	error |= mdb_dbi_open (transaction_a, path_a.c_str (), MDB_CREATE, &handle);
+	error |= mdb_dbi_open (transaction_a, ids_path.str ().c_str (), MDB_CREATE, &send_action_ids);
```

Comment:
I realize this pattern is used elsewhere, but this will try to open the second db even if the first one failed. Maybe we should change the pattern to `error = error || mdb_dbi_open ...` ?

Reply:
I don't think there's much harm to opening the second DB, and it's very unlikely to fail. I'd like to stick with precedent here.